webapp/channels/src/packages/mattermost-redux/src/utils/file_utils.ts

```diff
@@ -27,6 +27,10 @@ export function getFormattedFileSize(file: FileInfo): string {
     return `${bytes} B`;
 }
 
+export function formatBytes(bytes: number): string {
+    return getFormattedFileSize({size: bytes} as FileInfo);
```

Comment:
I'd probably modify `getFormattedFileSize` to accept a bytes number rather than the whole file info which can avoid this type assertion.